internal/commands/app/create.go

```diff
@@ -20,6 +20,10 @@ import (
 	"github.com/briandowns/spinner"
 )
 
+var (
+	noArgsDefaultValueTemplate = "<prompt-templates>"
```

Comment:
this is the default value of the --template flag when a user declares it without any args